docs/quickstart.rst

```diff
@@ -1,6 +1,166 @@
 Quickstart
 ==========
 
+Welcome to the *Septentrion* quickstart documentation.
+
+In this tutorial, you will learn how to write migrations in the expected format, how to visualize them and how to run
+them against a PostgreSQL database.
+
+Prepare your database
+---------------------
+
+If you already have a PostgreSQL database around, make sure to note the connection
+parameters. Otherwise, we'll create one together with Docker_:
+
+.. _Docker: https://docs.docker.com/
+
+.. code-block:: console
+
+    $ docker run --detach --rm -p 5432:5432 --name septentrion_db postgres
```

Comment:
Perhaps, either if it's a tutorial we want a persistent database, not use the `--rm` parameter

Reply:
The problem is that subsequent calls to `run` will still spawn different dockers, so running without --rm will not help a lot, afaik.

Reply:
We added a note to clarify the docker behaviour, so that people are not surprised when the database is empty after a reboot.